pages/index.tsx

```diff
@@ -162,14 +162,14 @@ export const HomePage: NextPage = () => {
         <CreateListAlert selectedService={selectedResource} />
         <CreateListDrawer />
       </CreateListProvider>
-      <Box px="112px" py="96px">
+      <Box px="10%" py="96px" minWidth="800px">
```

Comment:
Any particular reason for the switch to 10% padding?

If you look at your screenshots, all these `minWidth="800px"` are forcing the rest of the page to extend well beyond the navbar at the top - it doesn't make sense to have the `minWidth` be larger than the device width.

Reply:
looking into the fontsize stuff, but I'm a little confused why the pictures end up looking kind of okay (like is my chrome developer tool somehow not capturing the fact that there would be overflow if you were actually looking at it on a phone?)

Reply:
The dev tools are zooming out so you can see the whole page so maybe doesn't look like overflow, but it's not inaccurately capturing what's going on. If you drag the dev tools over so they take up more of the screen, maybe the overflow is more obvious.

<img width="1440" alt="Screenshot 2023-06-22 at 11 06 52 PM" src="https://github.com/MutualAidNYC/services-lists/assets/54749942/9a761fc4-34d0-4550-817d-c7c3505dd17c">